pyphare/pyphare/pharesee/phare_vtk/base.py

```
@@ -0,0 +1,108 @@
+
+    global_bounds = [0.0] * 6
+    bbox.GetBounds(global_bounds)
+    return global_bounds
+
+
+class VtkFile:
+    @staticmethod
+    def _phases():
+        return [surface_filter, composite_data_geometry_filter, poly_data_mapper]
+
+    def __init__(self, filename, time=None, array_name="data", phases=_phases()):
```

Ruff (0.14.11)

Do not perform function call `_phases` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)